requester/requester.go

```diff
@@ -87,9 +87,10 @@ type Work struct {
 	// Writer is where results will be written. If nil, results are written to stdout.
 	Writer io.Writer
 
+	init    sync.Once
```

Comment:
What about s/init/initOnce to make it clear it is a sync.Once?

Reply:
done